### PR TITLE
Fix revIDEGroupObjects dependency on object selection

### PR DESCRIPTION
**Pull request by montegoulding**

### CODE REVIEW

File: Toolset/libraries/revidelibrary.8.livecodescript [hunk 1]
```diff
@@ -3535,25 +3535,40 @@ on revIDEActionGroupObjects
...
-   // PM-2016-04-07: [[ Bug 17301 ]] Use the "group" command (without an objectList)
-   // so as to group the selection in layer order
-   group
    
-   put the long id of the owner of tObj into tGroupID
-   set the layer of tGroupID to tLayer
+   if pObjectIDs ends with return then
+      delete char -1 of pObjectIDs
+   end if
+
```
Comment (livecodeali) [inline Toolset/libraries/revidelibrary.8.livecodescript:3564]: Can't you just do

```
filter pObjectIDs without empty
```

or something here?

Comment (montegoulding) [inline Toolset/libraries/revidelibrary.8.livecodescript:3564]: yes of course... I'll fix

File: Toolset/libraries/revidelibrary.8.livecodescript [hunk 2]
```diff
@@ -3534,26 +3534,60 @@ on revIDEActionGroupObjects
...
+   end if
+   
+   local tChar
+   repeat with tChar = the number of chars of pObjectIDs down to 1
+      if char tChar of pObjectIDs is return then
+         delete char tChar of pObjectIDs
+      else
+         exit repeat
+      end if
+   end repeat
```
Comment (livecodeali) [inline Toolset/libraries/revidelibrary.8.livecodescript]: Doesn't the filter render this repeat loop redundant?

File: Toolset/libraries/revidelibrary.8.livecodescript [hunk 3]
```diff
@@ -3534,26 +3534,60 @@ on revIDEActionGroupObjects
...
-   // so as to group the selection in layer order
-   group
+Description:
+Group a list of objects in the order they are layered in
+
+Parameters:
+objectList (string): A return delimited list of long object IDs
+
+The result:
+The new group long ID or empty in the case the group could not becreated
```
Comment (livecodeali) [inline Toolset/libraries/revidelibrary.8.livecodescript]: You're missing a space in becreated

File: Toolset/libraries/revidelibrary.8.livecodescript (hunk 2 above)
Comment (montegoulding) [inline Toolset/libraries/revidelibrary.8.livecodescript]: Yes of course it does! GitHub really needs a facepalm reaction ;-) hacking away at autocomplete on the flights so I'll sort this PR out when I'm home.

Comment (livecodeali) [inline Toolset/libraries/revidelibrary.8.livecodescript]: Awesome, look forward to the results of that 😀

**Pull request closed without merging**